CHANGELOG.md

```diff
@@ -9,7 +9,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+
+### Features:
+
+### Bug fixes:
+
+### Internal:
+
```

Comment:
```suggestion
```

Reply:
Nah this is actually how we want it.

Reply:
It's awful and not how we had it previously